python/default-security-group-attached

```diff
@@ -0,0 +1,81 @@
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+# 
+# Description: Check that default security groups are not attached to EC2,RDS,ELB.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: EC2:NetworkInterfaces
```

Comment:
This should be singular --> `EC2:NetworkInterfaces`

Reply:
@msambol  thanks. will fix it